app/public/ai/ai.css

```diff
@@ -1,3 +1,64 @@
 /* For stylings used only on the AI's screen */
 
 
+/* General button styles, as provided */
```

Comment:
Are these stylings really "general" and shared for both players' pages? If so, let's move them to `gameScreen.css`